pip/requirements.txt

```diff
@@ -214,8 +214,9 @@ notebook==6.5.3
 notebook-shim==0.2.2
 #numba==0.56.4
 numexpr==2.8.4
+# setuptools version <64 is needed by numpy: https://github.com/pypa/setuptools/issues/3549
```

Comment:
@smuzaffar I think this warning should be in py3-setuptools.spec

Reply:
I agree, can you please update it. Also see if you can squash the commit.  there are too many commits for trying something and then reverting it :-(